cf-templates/main.yaml

```diff
@@ -127,11 +127,18 @@ Parameters:
     Type: String
     Default: "observability-cluster"
 
+  ######## OSI Variables ########
+  OSIPipelineBaseName:
+    Description: "Please enter the Base name for OSI pipelines "
+    Type: String
+    Default: "osi-pipeline"
+
+
   ######## General ########
   S3BucketName:
     Description: "Please enter the bucket name for the dependencies CloudFormation templates"
     Type: String
-    Default: "observability-with-amazon-opensearch"
+    Default: "ws-assets-prod-iad-r-iad-ed304a55c2ca1aee.s3.us-east-1.amazonaws.com/1abb648b-2ef8-442c-a731-efbcb69c1e1e"
```

Comment:
I would like to request to return the default value.

Reply:
These changes are necessary to use the bucket managed by the workshop.